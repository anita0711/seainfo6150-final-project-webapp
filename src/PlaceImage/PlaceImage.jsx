import React from "react";
import styles from "./PlaceImage.module.css";

const PlaceImage = ({ img1, img2, img3, img4, title }) => {
  return (
    <div className={styles.image}>
      <img src={img1} alt={title} />
    </div>
  );
};

export default PlaceImage;
