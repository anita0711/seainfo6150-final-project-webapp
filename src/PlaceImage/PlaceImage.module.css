/*.image {
height: 200px;
  width: 300px;
  border: 15px;
  align-items: center;
  justify-content: center;
}*/

.image {
  width: 80%;
  height: 200px;
  border-radius: 20px;
  margin: 0px;
  border: 5px solid #ccc;
  box-shadow: 0px 0px 20px black;
}

.image:hover {
  box-shadow: 10px 10px 20px black;
  cursor: pointer;
}
