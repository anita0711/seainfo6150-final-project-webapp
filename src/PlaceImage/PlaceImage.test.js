import React from "react";
import { render } from "@testing-library/react";
import PlaceImage from "./PlaceImage.jsx";

it("renders the PlaceImage component correctly", () => {
  const { container } = render(
    <PlaceImage
      img1="https://www.planetware.com/photos-large/USCA/california-san-francisco-golden-gate-bridge.jpg"
      title="San Francisco"
    />
  );
  expect(container).toMatchSnapshot();
});
