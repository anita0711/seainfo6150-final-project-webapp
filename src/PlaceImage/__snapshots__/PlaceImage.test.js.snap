// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the PlaceImage component correctly 1`] = `
<div>
  <img
    alt="San Francisco"
    class="image"
    src="https://www.planetware.com/photos-large/USCA/california-san-francisco-golden-gate-bridge.jpg"
  />
</div>
`;
